Elementary-Celebrity-Recognition-My-Dear-Watson/SwiftCloudWorkshop2.swift

```diff
@@ -38,7 +40,7 @@ import VisualRecognitionV3
 ///                ]
 ///            }
 ///
-func main(args: [String:Any]) -> [String:Any] {
+func main(args: [String: Any]) -> [String: Any] {
```

Comment:
Oh, good catch. Surprisingly, IBM Bluemix uses that form as the template example. I'll let the IBM folks know that they should update their template. 😉